.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+
+      - name: Release with Maven
+        run: mvn clean -Dusername=$GIT_USER_NAME -Dpassword=GITHUB_TOKEN release:prepare release:perform
+        env:
+          GIT_USER_NAME: acrolinx-opensource
```

Comment:
what is the advantage of declaring user name and email as env variables over hardcoding them in line 24 above?

Reply:
No advantage, just makes it easier for me to read, but that is relative.

Reply:
I think if we only use them once, I would prefer to inline them

Reply:
I just checked, the username is used twice, the email is used once. Should I still inline them?

Reply:
If it is used more than once, then having an env variable is good to avoid redundancy.
GIT_USER_EMAIL is only used once, so we could inline it.